db/seeds.rb

```diff
@@ -22,26 +22,58 @@
 Worcester
 Purvis]
 
-name = ['Del Taco',
-    'Zaxbys',
-    'Popeyes Louisiana Kitchen',
-    'El Pollo Loco',
-    'Chipotle Mexican Grill',
-    'Little Caesars',
-    'Papa Johns',
-    'Dominos Pizza',
-    'Steak n Shake',
-    'Pizza Hut',
-    'Dunkin Donuts',
-    'Fatburger',
-    'Long John Silvers',
-    'Jamba Juice',
-    'Quiznos',
-    'Wendys',
-    'Tim Hortons',
-    'In-N-Out Burger',
-    'Wingstop',
-    'Dairy Queen']
+name = [
+    "Molly Ward Gardens",
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.